crates/stateless/src/validation.rs

```diff
@@ -133,6 +137,31 @@ pub fn stateless_validation<ChainSpec, E>(
 where
     ChainSpec: Send + Sync + EthChainSpec + EthereumHardforks + Debug,
     E: ConfigureEvm<Primitives = EthPrimitives> + Clone + 'static,
+{
+    stateless_validation_with_trie::<StatelessTrie, ChainSpec, E>(
+        current_block,
+        witness,
+        chain_spec,
+        evm_config,
+    )
+}
```

Comment:
This wrapper method means that this does not break downstream users who were using the previous API.